.github/workflows/main.yml

```diff
@@ -54,4 +54,9 @@ jobs:
       - name: Go Generate
         run: |
           go generate ./...
-          git diff --quiet || (echo 'generated go files are not up to date, check go generate, go.sum and go.mod' ; git diff ; exit 1)
\ No newline at end of file
+          git diff --quiet || (echo 'generated go files are not up to date, check go generate, go.sum and go.mod' ; git diff ; exit 1)
+      - name: get Flamalyzer
```

Comment:
I don't think we need to get it first (see above on misspell and ineffassign)

Reply:
I thought the same but I get an error if I remove it 
```
no required module provides package flamingo.me/flamalyzer; to add it: go get flamingo.me/flamalyzer
```

Reply:
another idea?